onnxruntime/core/framework/tensorprotoutils.cc

```diff
@@ -31,6 +31,7 @@ TensorProto ToTensor<onnxruntime::MLFloat16>(const onnxruntime::MLFloat16& value
   TensorProto t;
   t.set_data_type(TensorProto_DataType_FLOAT16);
   t.add_int32_data(value.val);
+  t.add_dims(1);
```

Comment:
>t.add_dims(1); [](start = 2, length = 14)

why do we add dims for ToTensor(<single value>) but not for ToTensor(<multiple values>)? would it be better to be consistent? also should ONNX_NAMESPACE::ToTensor be updated too?